src/utils/helpers.ts

```diff
@@ -128,6 +128,29 @@ export const isAllowToEditContent = (
   }
 }
 
+export const isAllowToDeleteContent = (doc: IEditableDoc, user?: IUser) => {
+  if (!user) {
+    return false
+  }
+
+  if (isObservableObject(user)) {
+    user = toJS(user)
+  }
+
+  const roles =
+    user.userRoles && Array.isArray(user.userRoles) ? user.userRoles : []
```

Comment:
`Array.isArray` accepts falsy, truthy, objects, NaN... you can call it directly

```suggestion
  const roles =
    Array.isArray(user.userRoles) ? user.userRoles : []
```